x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -158,14 +214,16 @@ export const PolicyList = memo(() => {
         dataType: 'number',
```

Comment:
you probably should remove this since you are now rendering a component and not a `number`